metadata_annotation.ipynb

```diff
@@ -0,0 +1,2307 @@
+{
```

Comment:
<p>What is a pool here? A well? A sequencing lane?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/pertpy-tutorials/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>